app/controllers/staff/proposals_controller.rb

```diff
@@ -1,6 +1,8 @@
 class Staff::ProposalsController < Staff::ApplicationController
   before_action :enable_staff_program_subnav
   before_action :set_proposal_counts, only: [:index, :show, :selection]
+  before_action :skip_policy_scope
```

Comment:
It's worth verifying but I believe without this it was removing proposals from the program list.